src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -137,6 +137,20 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * # Ok
+ * messages.properties
+ * messages.translations
+ *
+ * # Violation: extension without translations or properties
+ * messages.txt
```

Comment:
we do not report violation on files that not a target for Check, please prove by CLI that it works like this. I  sure it is not working like you show.